packages/integration-tests/test/actions.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { Config, sleep } from '../../../common'
+import { JsonRpcProvider, Web3Provider } from '@ethersproject/providers'
+import { ganache } from '@eth-optimism/ovm-toolchain'
+import assert = require('assert')
+import * as fs from 'fs';
+
+import {
+  Contract, ContractFactory, providers, Wallet,
+} from 'ethers';
+
+let SimpleStorage
+const L1_USER_PRIVATE_KEY = Config.L1UserPrivateKey()
+const L2_USER_PRIVATE_KEY = Config.L2UserPrivateKey()
+const goerliURL = Config.L1NodeUrlWithPort()
+const optimismURL = Config.L2NodeUrlWithPort()
+const l1Provider = new JsonRpcProvider(goerliURL)
+const l2Provider = new JsonRpcProvider(optimismURL)
+const l1Wallet = new Wallet(L1_USER_PRIVATE_KEY, l1Provider)
+const l2Wallet = new Wallet(L2_USER_PRIVATE_KEY, l2Provider)
+const messengerJSON = JSON.parse(fs.readFileSync('contracts/build/iOVM_BaseCrossDomainMessenger.json').toString())
+const l2MessengerJSON = JSON.parse(fs.readFileSync('contracts/build/OVM_L2CrossDomainMessenger.json').toString())
```

Comment:
actually can probably chang this and the line above with `require('contracts/build/OVM_L2CrossDomainMessenger.json')`

Reply:
Unfortunately, I get linting or build errors from typescript when trying that syntax instead. There's some typescript config that can be changed to allow it, but I don't think that's necessary at the moment